Assets/LeapMotion/Core/Scripts/Utils/RuntimeGizmoManager.cs

```diff
@@ -531,8 +532,65 @@ public class RuntimeGizmoDrawer {
     /// <summary>
     /// Draws a wire gizmo sphere at the given position with the given radius.
     /// </summary>
-    public void DrawWireSphere(Vector3 center, float radius) {
-      DrawWireMesh(wireSphereMesh, center, Quaternion.identity, Vector3.one * radius * 2);
+    public void DrawWireSphere(Vector3 center, float radius, int numSegments = 32) {
+      //DrawWireMesh(wireSphereMesh, center, Quaternion.identity, Vector3.one * radius * 2);
+      DrawWireCircle(center, radius);
+
+      PushMatrix();
+
+      matrix = Matrix4x4.TRS(center, Quaternion.identity, Vector3.one)
+               * matrix;
+
+      var x = Vector3.right;
+      var y = Vector3.up;
+      var z = Vector3.forward;
+
+      DrawCameraAwareWireCircle(radius, y, x);
+      DrawCameraAwareWireCircle(radius, z, y);
+      DrawCameraAwareWireCircle(radius, x, z);
+
+      PopMatrix();
+    }
+
+    public void DrawCameraAwareWireCircle(float radius,
+                                          Vector3 normal, Vector3 radialStartDir,
+                                          int numSegments = 32) {
+
+      var origColor = this.color;
+
+      var camPos = matrix.inverse.GetPose().Then(Camera.main.transform.ToPose()).position;
```

Comment:
I'd want this method to perform some partially correct action if there is no main camera, that way things don't throw a null reference for an unknown reason.  (Also might want to look at if Camera.current would work better instead?)

Reply:
will fix